code/game/objects/items/cards_ids.dm

```diff
@@ -675,7 +678,7 @@
 	if(!alt_click_can_use_id(user))
 		return NONE
 	if(registered_account.account_debt)
-		var/choice = tgui_alert(user, "Choose An Action", "Bank Account", list("Withdraw", "Pay Debt"))
+		var/choice = tgui_alert(user, "Choose An Action", "Bank Account", list("Pay Debt"))
```

Comment:
^ Mark your edits ^
For this one copy the old line and have it in a comment to the right as the original, here's your example:

var/choice = tgui_alert(user, "Choose An Action", "Bank Account", list("Pay Debt")) /// LETHALSTATION EDIT CHANGE - ORIGINAL - var/choice = tgui_alert(user, "Choose An Action", "Bank Account", list("Withdraw", "Pay Debt"))